travis/Dockerfile-qt5-travis

```diff
@@ -0,0 +1,15 @@
+FROM openms/contrib:qt5-v2.0
+
+WORKDIR /
+ADD . /OpenMS
+RUN cd /OpenMS 
```

Comment:
Yes it would be the ADD command. Travis does a checkout of the sources in the beginning and you can include directories relative to the Dockerfile (which I download into the source folder for now)